.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 lib
 *.log
 src/example.ts
+.idea
```

Comment:
Can you remove this change? I believe our general stance at the moment is IDE-specific ignores shouldn't go in the committed `.gitignore`.

Reply:
Will do